charts/netdata/templates/daemonset.yaml

```diff
@@ -158,8 +158,20 @@ spec:
             - name: os-release
               mountPath: /host/etc/os-release
             {{- range $name, $config := .Values.child.configs }}
-            {{- if $config.enabled }}
-            - name: config
+            {{- if and $config.enabled (eq $config.storedType "configmap") }}
+            - name: configmap
+              mountPath: {{ $config.path }}
+              subPath: {{ $name }}
+            {{- else if and $config.enabled (eq $config.storedType "secret") }}
```

Comment:
I think we need here only 2 conditions: it is either `configmap` or `secretmap`, e.g.:

```yaml
{{- if $config.enabled }}
            - name: {{ ternary "configmap" "secretmap" (ne $config.storedType "secret") }}
              mountPath: {{ $config.path }}
              subPath: {{ $name }}
{{- end }}
```

True for all deployments.